templates/deno/app/entry.client.tsx

```diff
@@ -1,5 +1,5 @@
-import React from "react";
-import ReactDOM from "react-dom";
 import { RemixBrowser } from "@remix-run/react";
+import * as React from "react";
```

Comment:
Why is `* as React` used here instead of just `React`?

Reply:
This is the recommended way of importing `react`

Reply:
Is that recommendation from the React docs? Or from us? or elsewhere?

Reply:
The React team is recommending it since v17 I think.
They even have a codemod for it if I'm not mistaken + there's an issue about deprecating default export in their repo.

Don't have the time to search for them currently, but I know I @kentcdodds has updated these imports in his courses in the past.

Reply:
Ok yes I recall that using `* as React` means that `tsconfig.json` doesn't need to be configured with `allowSyntheticDefaultImports`. Agreed, these should be `* as React` 👍